setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup
 from glob import glob
 
-__version__ = "0.1.0"
+__version__ = "2021.03"
 
 classes = """
     Development Status :: 3 - Alpha
```

Comment:
Is this still Alpha?

Reply:
Excellent point, moving to: `Development Status :: 5 - Production/Stable` as all this have been deployed/in-used for a few year now.